CONTRIBUTING.md

```diff
@@ -183,13 +183,24 @@ One thing you will need to do only for your first pull request in Netdata is to
 
 #### PR title guidelines
 
-All verbs in PR titles must be in the past tense. E.g. "Added" instead of "Adds", "Fixed" instead of "Fixes".
+- All PR titles and Commit messages _must_ follow the [Imperative Mood](https://en.wikipedia.org/wiki/Imperative_mood)
+- Be no more than ~50 characters (_longer description in the PR_)
```

Comment:
Yes! I would stress the point that the PR description MUST contain sufficient information regarding the content of the PR, including area/component, tests done, etc.

Reply:
Done